### DISCUSSION TITLE
Quick Import - AIRTABLE (Cannot read property 'length' of undefined)

### DISCUSSION THREAD

**Discussion by kmataeh23** [nocodb/nocodb#3925] [General]
Hi Nocodb team. I am trying to import an air table to noco using its api key and shared base ID but am getting a "cannot read property 'length' of undefined" error in the logs section. Any advice on a workaround for this error?
<img width="1045" alt="read property length undefined" src="https://user-images.githubusercontent.com/70729030/193707852-11b14a29-c632-4f6b-9e67-36d7b07da960.png">

**Comment by dstala** [comment]
@kmataeh23 
You can DM me additional details on discord (@dstala). If you can share redacted schema, that would be helpful for us to investigate

**Reply by kmataeh23** [reply]
Hey @dstala . Here is a picture of some redacted schema in our AirTable. We appear to get a warning from Airtable stating "This base has gone over the record limit. To keep adding upgrades request an upgrade." I'm not sure if this is what is causing the "cannot read property length error, but perhaps? Thank you for any help!
<img width="1386" alt="possible source of length overflow" src="https://user-images.githubusercontent.com/70729030/193847698-48e139cd-b268-40cc-8305-2f1d4b8c8aee.png">

**Reply by mertmit** [reply]
@kmataeh23 this is unlikely to be related with this, we need error logs from your instance (if you are on docker use: 'docker container logs container_name') or `shared base ID` to reproduce and find what is causing this issue.

**Reply by kmataeh23** [reply]
@mertmit  Sorry for the hold up. Here are my docker logs showing the instance where this issue occurred.
<img width="1190" alt="docker logs" src="https://user-images.githubusercontent.com/70729030/194645194-dc0d7c37-dfb5-400d-896d-29f9b3c2f76b.png">

**Comment by kmataeh23** [comment]
@mertmit 

I am also getting an error "Some internal issue occurred" when I try to upload some of the raw csv files instead. Here are my docker logs from the instance
<img width="1191" alt="csv file error" src="https://user-images.githubusercontent.com/70729030/194971514-2adb5e29-ea9e-4e46-bfcb-9311a794f4fc.png">
